Dockerfile

```diff
@@ -120,6 +120,13 @@ ENV CASSANDRA_CLUSTER=false \
 # Configure search nodes. Empty string falls back to search being done on the nodes themselves.
 ENV PEGA_SEARCH_URL=
 
+# Configure hazelcast. By default, hazelcast runs in embedded mode.
+Env CLIENT_MODE=false \
+    ON_PEGA_K8S= \
```

Comment:
Why is this needed?

Reply:
There are different types of client server model , pega cuttyhunk , on-premise and K8s deployment. So to indicate that this client mode is for K8s deployment , we need to set this value <Environment name="prconfig/Pega-RULES/onPegaK8s" value="{{ .Env.ON_PEGA_K8S }}" type="java.lang.String"/> to true.

Reply:
on_pega_k8s is used by hazelcast cluster module, if this config is true then it will use kubernetes based discovery between hazelcast clients and servers. We have cuttyhunk based client-server model as well so we are using this setting to differentiate between aws based discovery and kubernetes based discovery for hazelcast

Reply:
In that case, can we change the name to reflect what it is used for?  e.g. HZ_DISCOVERY_K8S

Reply:
Sure , I will rename this parameter to `HZ_DISCOVERY_K8S`